<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsDialog</class>
 <widget class="QDialog" name="ControlsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkPen">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkAllGen">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_aa">
       <property name="text">
        <string>&amp;Antialiasing</string>
       </property>
       <property name="buddy">
        <cstring>checkAntialiasing</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_approx_maxx_gen">
       <property name="text">
        <string>Approx &amp;max generatoin</string>
       </property>
       <property name="buddy">
        <cstring>spinApproxMaxGen</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_approx_bbox_gen">
       <property name="text">
        <string>Approx &amp;bbox generation</string>
       </property>
       <property name="buddy">
        <cstring>spinApproxBboxGen</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkAntialiasing">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_pen">
       <property name="text">
        <string>Fancy &amp;pen</string>
       </property>
       <property name="buddy">
        <cstring>checkPen</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="generations">
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_approx">
       <property name="text">
        <string>Appro&amp;x algorithm</string>
       </property>
       <property name="buddy">
        <cstring>checkApprox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_all_gen">
       <property name="text">
        <string>A&amp;ll generations</string>
       </property>
       <property name="buddy">
        <cstring>checkAllGen</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_generations">
       <property name="text">
        <string>&amp;Generations</string>
       </property>
       <property name="buddy">
        <cstring>generations</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_approx_max_vertices">
       <property name="text">
        <string>Approx max &amp;vertices</string>
       </property>
       <property name="buddy">
        <cstring>spinApproxMaxVertices</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkApprox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinApproxBboxGen">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinApproxMaxGen">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="spinApproxMaxVertices">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="singleStep">
        <number>1000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="pointCoordsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_x">
        <property name="text">
         <string>&amp;x</string>
        </property>
        <property name="buddy">
         <cstring>edit_x</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="edit_x">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_y">
        <property name="text">
         <string>&amp;y</string>
        </property>
        <property name="buddy">
         <cstring>edit_y</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="edit_y">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>generations</tabstop>
  <tabstop>checkAntialiasing</tabstop>
  <tabstop>checkPen</tabstop>
  <tabstop>checkAllGen</tabstop>
  <tabstop>checkApprox</tabstop>
  <tabstop>spinApproxBboxGen</tabstop>
  <tabstop>spinApproxMaxGen</tabstop>
  <tabstop>spinApproxMaxVertices</tabstop>
  <tabstop>edit_x</tabstop>
  <tabstop>edit_y</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
